### PR TITLE
Connect Validation Result List Items (EXPOSUREAPP-8329)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Connect Data to ViewHolder

### Testing
- Scan RC, TC, VC and validate them against the country rules , you should see the relevant Result screen displaying the right information 

Merge after: https://github.com/corona-warn-app/cwa-app-android/pull/3695

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml [hunk 1]
```diff
@@ -188,44 +188,68 @@
...
+    <string name="validation_rules_technical_failure_header_title">"Technische Prüfung"</string>
     <!-- XTXT: Validation Rule Technical Failure Header Subtitle  -->
-    <string name="validation_rules_technical_failure_header_subtitle">Folgende Regeln sind nicht erfüllt:</string>
+    <string name="validation_rules_technical_failure_header_subtitle">"Folgende Regeln sind nicht erfüllt:"</string>
     <!-- XTXT: Validation Rule Technical Failure - Date Format  -->
-    <string name="validation_rule_technical_error_date_format">Das Zertifikat muss dem erwarteten Datenformat entsprechen.</string>
+    <string name="validation_rule_technical_error_date_format">"Das Zertifikat muss dem erwarteten Datenformat entsprechen."</string>
     <!-- XTXT: Validation Rule Technical Failure - Date Expired  -->
-    <string name="validation_rule_technical_error_date_expired">Das Zertifikat darf nicht abgelaufen sein.</string>
+    <string name="validation_rule_technical_error_date_expired">"Das Zertifikat darf nicht abgelaufen sein."</string>
```
Comment (ChristinaTsigganos) [inline Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml:215]: This message needs to be replaced by the new message "Das technische Ablaufdatum darf nicht überschritten sein.
" as described in Figma: https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.2?node-id=25537%3A21835

File: Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml [hunk 2]
```diff
@@ -195,41 +195,61 @@
...
+    <string name="validation_rules_technical_failure_header_title">"Technische Prüfung"</string>
     <!-- XTXT: Validation Rule Technical Failure Header Subtitle  -->
-    <string name="validation_rules_technical_failure_header_subtitle">Folgende Regeln sind nicht erfüllt:</string>
+    <string name="validation_rules_technical_failure_header_subtitle">"Folgende Regeln sind nicht erfüllt:"</string>
     <!-- XTXT: Validation Rule Technical Failure - Date Format  -->
-    <string name="validation_rule_technical_error_date_format">Das Zertifikat muss dem erwarteten Datenformat entsprechen.</string>
+    <string name="validation_rule_technical_error_date_format">"Das Zertifikat muss dem erwarteten Datenformat entsprechen."</string>
     <!-- XTXT: Validation Rule Technical Failure - Date Expired  -->
-    <string name="validation_rule_technical_error_date_expired">Das Zertifikat darf nicht abgelaufen sein.</string>
+    <string name="validation_rule_technical_error_date_expired">"Das Zertifikat darf nicht abgelaufen sein."</string>
```
Comment (ChristinaTsigganos) [inline Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml:219]: same here please

**Review by ChristinaTsigganos — CHANGES_REQUESTED** [review]
Other than the text change to replace old by new message this PR is UA approved

File: Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml [hunk 3]
```diff
@@ -202,41 +202,74 @@
     <string name="validation_rules_result_valid_result_country_and_time">Einreise nach %1$s am %2$s, geprüft am %3$s</string>
 
     <!-- XTXT: Validation Rule Result Valid - Result  -->
-    <string name="validation_rules_result_valid_result_title">Ihr Zertifikat ist im gewählten Land gültig.</string>
+    <string name="validation_rules_result_valid_result_title">"Ihr Zertifikat ist im gewählten Land gültig."</string>
```
Comment (NataliaLemmerth) [inline Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml:205]:
```suggestion
    <string name="validation_rules_result_valid_result_title">"Ihr Zertifikat ist im gewählten Land gültig"</string>
```

File: Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml [hunk 4]
```diff
@@ -202,41 +202,74 @@
...
     <!-- XTXT: Validation Rule Result Cannot be checked  - Result  -->
-    <string name="validation_rules_result_cannot_be_checked_result_title">Ihr Zertifikat konnte nicht vollständig geprüft werden.</string>
+    <string name="validation_rules_result_cannot_be_checked_result_title">"Ihr Zertifikat konnte nicht vollständig geprüft werden."</string>
```
Comment (NataliaLemmerth) [inline Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml:207]:
```suggestion
    <string name="validation_rules_result_cannot_be_checked_result_title">"Ihr Zertifikat konnte nicht vollständig geprüft werden"</string>
```

File: Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml [hunk 5]
```diff
@@ -202,41 +202,74 @@
...
     <!-- XTXT: Validation Rule Result Not Valid - Result  -->
-    <string name="validation_rules_result_not_valid_result_title">Ihr Zertifikat ist im gewählten Land nicht gültig.</string>
+    <string name="validation_rules_result_not_valid_result_title">"Ihr Zertifikat ist im gewählten Land nicht gültig."</string>
```
Comment (NataliaLemmerth) [inline Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml:209]:
```suggestion
    <string name="validation_rules_result_not_valid_result_title">"Ihr Zertifikat ist im gewählten Land nicht gültig"</string>
```

**Review by NataliaLemmerth — CHANGES_REQUESTED** [review]
Small text changes. Everything else is fine.

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by mtwalli — DISMISSED** [review]
Lgtm

**Review by BMItr — APPROVED** [review]
LGTM!
Basic Device Tests successful 👌 
image

**Review by mtwalli — APPROVED** [review]
LGTM

**Pull request merged**